hxd/res/BDFFont.hx

```diff
@@ -0,0 +1,436 @@
+package hxd.res;
+
+#if (haxe_ver < 4)
+import haxe.xml.Fast in Access;
+#else
+import haxe.xml.Access;
+#end
+
+import hxd.BitmapData;
+import haxe.Int32;
+
+/**
+ * Intermediate representation of a glyph. Only used while
+ * parsing a BDF font file.
+ */
+ class BDFFontChar{
```

Comment:
CodeStyle: All opening brackets should contain space before them (`expr {`)

Reply:
Do you also prefer I remove brackets from `case` or also ensure there is a space before them?
`case 'ENCODING': { ... }`

Reply:
Huh, didn't notice brackets on cases. But yes, better remove them, as they are completely unnecessary.

Reply:
Yes, I just like them for readability. Will remove them. Will always keep them in my own code :)